client/app/cloud/project/compute/infrastructure/virtualMachine/delete/cloud-project-compute-infrastructure-virtual-machine-delete.html

```diff
@@ -1,41 +1,58 @@
+            <button
+                type="button"
+                data-ng-click="$ctrl.cancel()"
+                class="oui-icon oui-icon-close_thin oui-modal__close-button"
+                data-translate-attr="{ 'aria-label': 'common_cancel' }"></button>
+        </div>
+    
+        <div class="oui-modal__body">
+    
+            <span class="oui-icon oui-icon-help_circle oui-icon_bicolor" aria-hidden="true"></span>
+    
+            <div class="oui-modal__body-text">
+                <h2 class="oui-modal__title" data-translate="cpcivm_delete_title"></h2>
+    
+                <div class="oui-modal__text">
+                    <p data-translate="cpcivm_delete_confirm"></p>
+    
+                    <dl>
+                        <dt data-translate="cpcivm_instance_name"></dt>
+                        <dd data-ng-bind=":: $ctrl.vm.name"></dd>
+                        <dt data-translate="cpcivm_instance_model"></dt>
+                        <dd data-ng-bind=":: $ctrl.vm.flavorTranslated"></dd>
+                        <dt data-translate="cpcivm_instance_ipv4"></dt>
+                        <dd data-ng-bind=":: $ctrl.vm.ipv4"></dd>
+                        <dt data-translate="cpcivm_instance_ipv6"></dt>
+                        <dd data-ng-bind=":: $ctrl.vm.ipv6"></dd>
+                    </dl>
+    
+                    <strong
+                        role="alert"
+                        data-ng-if="$ctrl.isMonthlyBilling"
+                        data-translate="cpcivm_delete_info"></strong>
+                </div>
             </div>
-            <p class="oui-modal__text"><strong data-translate="cpcivm_delete_confirm"></strong></p>
         </div>
-    </div>
-
-    <div class="oui-modal__footer">
-        <button type="reset" class="oui-button oui-button_secondary"
-            data-ng-click="$ctrl.cancel()">
-            <span data-translate="common_cancel"></span>
-        </button>
-        <button type="submit" class="oui-button oui-button_primary"
-            data-ng-click="$ctrl.backup()"
-            data-track-on="click"
-            data-track-name="VM-Delete"
-            data-track-click="action">
-            <span data-translate="common_confirm"></span>
-        </button>
-    </div>
-
-</form>
+    
+        <div class="oui-modal__footer">
+            <oui-button
+                type="reset"
+                data-on-click="$ctrl.cancel()"
+                data-variant="secondary"
+                data-text="{{:: 'common_cancel' | translate }}"
+                class="mr-3"></oui-button>
+            <button
+                type="submit"
+                data-ng-click="$ctrl.backup()"
+                data-track-on="click"
+                data-track-name="VM-Delete"
+                data-track-click="action"
+                class="oui-button oui-button_primary">
+                <span data-translate="common_confirm"></span>
+            </button>
+        </div>
+    
+    </form>
```

Comment:
https://github.com/ovh-ux/ovh-manager-cloud/blob/master/.editorconfig#L18

Indentation issue?

Reply:
For some reason EditorConfig was not installed on my VS Code 🤔